pro-dental-care-ui/src/app/patient/overview/patient-overview.component.html

```diff
@@ -0,0 +1,49 @@
+<main class="flex flex-col p-14 bg-white max-w-[940px] max-md:px-5">
+    <header class="flex flex-col max-w-full leading-tight w-[568px]">
+      <h1 class="tracking-tight font-[number:var(--sds-typography-heading-font-weight)] text-[color:var(--sds-color-text-default-default)] text-[length:var(--sds-typography-heading-size-base)] max-md:max-w-full">
+        Hello, Jane Doe
+      </h1>
+      <p class="mt-2 font-[number:var(--sds-typography-subheading-font-weight)] text-[color:var(--sds-color-text-default-secondary)] text-[length:var(--sds-typography-subheading-size-medium)] max-md:max-w-full">
+        This is your overview. Use the sidebar for more specific information.
+      </p>
+    </header>
+    <!-- Section for Overview Boxes -->
+    <section class="flex flex-wrap gap-12 items-center mt-11 w-full min-h-[316px] max-md:mt-10 max-md:max-w-full">
+      <!-- Appointment -->
+      <article class="overview-box">
+        <div class="icon-placeholder" aria-label="Appointment illustration">
+          <img src="assets/clock.png" alt="Appointment Icon" />
```

Comment:
In this and other places where you use images sources, please use the `ngSrc` attribute instead of src.
Also, for the source of the image you should use full paths such as `/assets/clock.png` with the leading forward slash.

Reply:
fixed it! if its all good, u can merge pull request! :)